cfroutesync/webhook/service_builder.go

```diff
@@ -36,8 +34,8 @@ func (b *ServiceBuilder) routeToServices(route models.Route, template Template)
 			},
 			Spec: ServiceSpec{
 				Selector: map[string]string{
-					b.PodLabelPrefix + "app_guid":     dest.App.Guid,
-					b.PodLabelPrefix + "process_type": dest.App.Process.Type,
+					"cloudfoundry.org/app_guid":     dest.App.Guid,
```

Comment:
I think it's a good idea to extract this to some constant to avoid typos in the future.